2019/clacks-solution/german/sentence_translations.json

```diff
@@ -622,7 +622,7 @@
  },
  {
   "input": "Remember, this rise over run slope of that constant momentum line was the negative square root of m1 over m2, which in this example is negative 10. ",
-  "translatedText": "Denkt daran, dass die Steigung dieser Linie mit konstantem Impuls die negative Quadratwurzel von m1 durch m2 war, die in diesem Beispiel minus 10 ist. ",
+  "translatedText": "Denkt daran, dass die Steigung dieser Linie mit konstantem Impuls die negative Quadratwurzel von m1 durch m2 ist, die in diesem Beispiel minus 10 ist. ",
```

Comment:
For better consistency I'd suggest putting `Denkt daran` into singular instead of the plural form.
E.g.: `Denk dran/Denk daran` or `Denke dran/Denke daran`